CHANGES.txt

```diff
@@ -1,4 +1,5 @@
 4.0-alpha5
+ * Improve token allocator (followup CASSANDRA-15600)
```

Comment:
Perhaps something more specific, referencing CASSANDRA-15877?

Reply:
Yes, sure, I forgot to finalize it, this was the intermidiate change to remind to not forget it (apparently this strategy didn't work :D )